src/Analysis/GWASResultsV2/GWASResultsContainer.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import Home from './Views/Home/Home';
+import Results from './Views/Results/Results';
+import Execution from './Views/Execution/Execution';
+import SharedContext from './Utils/SharedContext';
+import GetTableDataFromApi from './Utils/GetTableDataFromApi';
+import './GWASResultsContainer.css';
+
+const GWASResultsContainer = () => {
+  const [currentView, setCurrentView] = useState('home');
+  const [currentExecutionData, setCurrentExecutionData] = useState({});
+  const [currentResultsData, setCurrentResultsData] = useState({});
+  const [tableData, setTableData] = useState(GetTableDataFromApi());
+  const pollingIntervalinMilliseconds = 5000;
+
+  // API Polling:
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setTableData(GetTableDataFromApi());
+    }, pollingIntervalinMilliseconds);
+    return () => clearInterval(interval);
+  }, []);
```

Comment:
I think it makes sense to move this to Views/Home/Home.jsx or to Views/Home/HomeTable/HomeTable.jsx

Reply:
Good point! This has been move to Views/Home/Home.jsx